dashboard/models.py

```diff
@@ -263,7 +263,7 @@ def get_course_resource_type(self, course_id):
         :return:
         """
         try:
-            return list(self.values_list('resource_type', flat=True).distinct().filter(course_id=course_id))
+            return list(self.values_list('resource_type', flat=True).distinct().filter(resourceaccess__course_id=course_id))
```

Comment:
This is a good addition.

I've been thinking that we may need to be a little more sophisticated with identifying resource types.  MiVideo resources have at least two types, video and audio.  At some point may want to be specific about this.  I can't think of many good reasons, but knowing the specific type would allow us to show a more appropriate icon for the resource in the UI views.

Reply:
Yeah it's pretty generic right now, determined by just a few times. This was desired by the project team to just have a few top level types like **files, videos, reading.** 

I do think we could add a feature (maybe I even did where we could have a multi-select that would be like

```
Videos -> MiVideo
          Lecture Capture
          Zoom
```

Or something like that but not worth future proofing around this for now since this could relatively easily be added.